javascripts/discourse/api-initializers/discourse-topic-list-author.gjs

```diff
@@ -0,0 +1,76 @@
+import { eq } from "truth-helpers";
+import bodyClass from "discourse/helpers/body-class";
+import { withPluginApi } from "discourse/lib/plugin-api";
+import AuthorColumnContent from "../components/author-column-content";
+import AuthorColumnHeader from "../components/author-column-header";
+import CustomCategoriesLatestAvatar from "../components/custom-categories-latest-avatar";
+
+export default {
+  name: "discourse-topic-list-cards",
+  initialize() {
+    withPluginApi("1.39.0", (api) => {
+      api.renderInOutlet(
+        "latest-topic-list-item-topic-poster",
+        CustomCategoriesLatestAvatar
+      );
+
+      const site = api.container.lookup("service:site");
+
+      // TODO: (discourse.hbr-topic-list-overrides) remove the connector rendered below after the legacy topic list is removed from core
```

Comment:
```suggestion
      // TODO: (discourse.hbr-topic-list-overrides) remove the connector rendered below after the legacy topic list is removed from core
      // Search keyword: discourse.hbr-topic-list-overrides
```
to help us find it later

Reply:
I tagged the TODO with `discourse.hbr-topic-list-overrides`